modules/core/src/scenegraph/gltf/gltf-instantiator.js

```diff
@@ -155,15 +155,22 @@ export default class GLTFInstantiator {
     return loadedAttributes;
   }
 
-  createBuffer(bufferView, target) {
+  createBuffer(attribute, target) {
+    if (!attribute.bufferView) {
+      // Draco decoded files do not have a bufferView
```

Comment:
Interesting point. Maybe we should add new bufferViews, after decoding.

I removed the extension after decompression with the intention that the app wouldn't be able to tell that it was ever there, but here we are, adding special cases to handle it...

Reply:
Maybe.